Gravatar.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
```

Comment:
Chybí namespace.

Dal bych:

```suggestion
namespace Baraja\Gravatar;
```